site/tutorials/gettings_started.md

```diff
@@ -77,9 +77,9 @@ To show the data as a point, we can set the `mark` property to `point`.
 }
 </div>
 
-Now, we can see a point. In fact, Vega-Lite renders one point for each object in the array, but they are all overlapping.
+Now, it looks like we get a point. In fact, Vega-Lite renders one point for each object in the array, but they are all overlapping since we have not mapped data values to visual channels of the marks.
```

Comment:
I'd remove the `since we have not mapped data values to visual channels of the marks` because it confuses more than it helps. We explain later how to resolve the issue.